registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -251,6 +252,7 @@
     "sha256:3a8e859273778370962d8ad9e496d071b414202bc13aef8e06e2459c836b7449": ["v1.10.4"]
     "sha256:f588fbe6f73160981cb7ae8ed9d1e48ea0322ddf3ec28125532cb0af01d91ce5": ["v1.10.5"]
     "sha256:913e0938a27a02947a24becc6b409511d5bfb4f95c6d51eb4137243c93f00b11": ["v1.10.6"]
+    "sha256:a8f8975d97a7e1feaaf11368a92beb30dcbeedf1df844267840714f522660ad7": ["v1.10.7"]
```

Comment:
SHA `a8f8975d97a7` matches clusterctl v1.10.7 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:a8f8975d97a7e1feaaf11368a92beb30dcbeedf1df844267840714f522660ad7/details).